pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md

```diff
@@ -104,3 +104,47 @@ _Table 1: Resource Requirements_
 1. The names _ESXMgr_ and _CloudCache_ are abbreviated names for the VMs containing HDM components. These are created as part of the HDM deployment.
 2. The N in _Cluster (N)_ refers to the number of nodes in the cloud cluster.
 
+
+### Prepare Cloud for Deployment
+
+To deploy the HDM solution on  _IBM Cloud for VMware Solutions Shared_ you would
+need to
+
+* Get the _Organization Administrator_ access credentials to the VCD tenant portal.
+* Identify the organization ID and save for later use.
+* Create a new user with admin privilege.
+
+**Identify the oprganization ID**
+
+* Loging to the tenant portal
+* The Organization ID can be identified from the location in  _Figure 1_
+
+_Figure 1: Screenshot show how to identify Organization ID_
+![Organization Id](images/Organization-id.png?classes=content-img "Screenshot showing how to find organization id") 
+
+
+**Create new user with admin priveledges**
+
+We do not recommend using the org administrator credentials directly
+while deploying HDM. Instead we recommend that the steps to create a new user with 
```

Comment:
recommend **that** the steps - need to remove that.

Can we rephrase it as - Instead we recommend the creation of a new organization admin for HDM integration. We will refer to this new admin in the rest of the document.

Reply:
done